crossbundle/cli/src/commands/install/mod.rs

```diff
@@ -1,20 +1,48 @@
 pub mod bundletool;
+pub mod ndk;
+pub mod sdkmanager;
 
 use crate::error::Result;
 use clap::Parser;
 use crossbundle_tools::utils::Config;
 
-use self::bundletool::AndroidInstallCommand;
+use self::{bundletool::BundletoolInstallCommand, sdkmanager::SdkmanagerInstallCommand};
 
 #[derive(Parser, Clone, Debug)]
 pub enum InstallCommand {
-    Bundletool(AndroidInstallCommand),
+    Bundletool(BundletoolInstallCommand),
+    Sdkmanager(SdkmanagerInstallCommand),
 }
 
 impl InstallCommand {
     pub fn handle_command(&self, config: &Config) -> Result<()> {
         match self {
             InstallCommand::Bundletool(cmd) => cmd.install(config),
+            InstallCommand::Sdkmanager(cmd) => cmd.install(config),
         }
     }
 }
+
+/// Download jar file and save it in directory
+pub fn create_jar_file(
+    download_url: String,
+    jar_path: std::path::PathBuf,
+) -> crate::error::Result<()> {
+    let response = ureq::get(&download_url)
+        .call()
+        .map_err(crate::error::Error::DownloadFailed)?;
+
+    let mut out = std::fs::File::create(&jar_path).map_err(|cause| {
```

Comment:
I think we can write to created file right away with [write](https://doc.rust-lang.org/stable/std/io/trait.Write.html#tymethod.write).

Reply:
I think this is not so simple as it seems. Response struct doesn't implement std::io::write trait

Reply:
We have returned Response struct when ureq::get() function was invocated.